### PR TITLE
For-While loops were added

### PR DESCRIPTION
**Pull request by ilequeque**

### CODE REVIEW

File: Before-After-Python/After/for-loop.spy [hunk 1]
```diff
@@ -0,0 +1,5 @@
+x:int = 0
+for i:int int range(0, 10):
```
Comment (MirasTalaspayev) [inline Before-After-Python/After/for-loop.spy:2]: You got a typo. change 'int' to 'in'.

File: Before-After-Python/After/while-loop.spy [hunk 2]
```diff
@@ -0,0 +1,7 @@
+y:int = 2
+j:int = -4
+while j < 0 :
+	y=y*2
+	j=j+2
+
+printf(y)
```
Comment (MirasTalaspayev) [inline Before-After-Python/After/while-loop.spy:7]: change 'printf' to 'print'

**Comment by ilequeque** [comment]
Everything was fixed

File: Before-After-Python/After/for-while.spy [hunk 3]
```diff
@@ -0,0 +1,12 @@
+x:int = 0
+y:int = 2
+j:int = 10
+
+while j >= 0 :
+	for i:int int range(0, 5):
```
Comment (MirasTalaspayev) [inline Before-After-Python/After/for-while.spy:6]: again typo. change 'int' to 'in'

Comment (ilequeque) [reply Before-After-Python/After/for-while.spy]: fixed

**Review by MirasTalaspayev — APPROVED** [review]
good!

**Pull request merged**